app/views/catalog/_search_results.html.erb

```diff
@@ -48,7 +48,7 @@
       <div class="sidebar col-12 col-md-4">
         <!-- TODO: search mode toggle -->
 
-        <%= render Blacklight::Response::FacetGroupComponent.new(id: nil, title: 'Filters') do |component| %>
+        <%= render Facets::GroupComponent.new(id: nil, title: 'Filters') do |component| %>
```

Comment:
Would the implementation be easier if we split the facets into separate groups (collapsible + non-collapsible?) Maybe that's the explicit identification I'm suggesting in https://github.com/sul-dlss/SearchWorks/pull/5165/files#r2157474808 🤷‍♂️